sound/Music.java

```diff
@@ -4,32 +4,32 @@
 import javax.sound.midi.MidiChannel;
 import javax.sound.midi.MidiSystem;
 import java.util.Iterator;
-
+import imagelab.ImgProvider;
 /**
  * Render sound using the javax.sound.midi package.
  * @author Dr. Jody Paul
  * @version 1.2
  */
 public class Music {
-    /** Standard Velocity */
+    /** Standard Velocity. */
     public static final int STD_VELOCITY = 64;
-    /** Standard Duration */
+    /** Standard Duration. */
     public static final int STD_DURATION = Note.DE / 2;
-    /** Standard Instrument */
+    /** Standard Instrument. */
     public static final int STD_INSTRUMENT = Note.Vibes;
-    /** Synthesizer reference */
-    Synthesizer synth;
-    /** Synthesizer channels */
-    MidiChannel[] channels;
-    /** code for default instrument */
-    int instrument;
-    /** Default note velocity */
-    int velocity = STD_VELOCITY;
-    /** Default number of Channels */
-    int numChannels = 1;
-    
-    /**
-     * Construct Music using Note.Vibes instrument.
+    /** Synthesizer reference. */
+    private Synthesizer synth;
+    /** Synthesizer channels. */
+    private MidiChannel[] channels;
+    /** Code for default instrument. */
+    private int instrument;
+    /** Default note velocity. */
+    private int velocity = STD_VELOCITY;
+    /** Default number of Channels. */
+    private int numChannels = 1;
+
+    /**
+     * Construct Music without parameters.
```

Comment:
This is an uninformative comment.  The header itself says it is a Music constructor and that there are no parameters.  What was the reason for the change?  (If it was that "Note.Vibes instrument" was overly specific, then perhaps refer to the class constant STD_INSTRUMENT.)